src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="day-1 calendar__day calendar__day--start-day--mon"></div>
+      <div class="day-2 calendar__day"></div>
+      <div class="day-3 calendar__day"></div>
+      <div class="day-4 calendar__day"></div>
+      <div class="day-5 calendar__day"></div>
+      <div class="day-6 calendar__day"></div>
+      <div class="day-7 calendar__day"></div>
+      <div class="day-8 calendar__day"></div>
+      <div class="day-9 calendar__day"></div>
+      <div class="day-10 calendar__day"></div>
+      <div class="day-11 calendar__day"></div>
+      <div class="day-12 calendar__day"></div>
+      <div class="day-13 calendar__day"></div>
+      <div class="day-14 calendar__day"></div>
+      <div class="day-15 calendar__day"></div>
+      <div class="day-16 calendar__day"></div>
+      <div class="day-17 calendar__day"></div>
+      <div class="day-18 calendar__day"></div>
+      <div class="day-19 calendar__day"></div>
+      <div class="day-20 calendar__day"></div>
+      <div class="day-21 calendar__day"></div>
+      <div class="day-22 calendar__day"></div>
+      <div class="day-23 calendar__day"></div>
+      <div class="day-24 calendar__day"></div>
+      <div class="day-25 calendar__day"></div>
+      <div class="day-26 calendar__day"></div>
+      <div class="day-27 calendar__day"></div>
+      <div class="day-28 calendar__day"></div>
+      <div class="day-29 calendar__day"></div>
+      <div class="day-30 calendar__day calendar__day--month-length--30"></div>
```

Comment:
the same:
![image](https://user-images.githubusercontent.com/86923089/194704670-b8d55a2e-bd05-46fe-98f4-04a182f19886.png)